packages/verification-vue-component/src/components/CertifactionVerification.vue

```diff
@@ -1,23 +1,35 @@
 <template>
-    <div class="certifaction-verification">
-        <VerificationDemo v-if="demo !== false" @verify-demo="verifyDemo" @dragging-demo-doc="onDraggingDemoDoc"/>
-
-        <div v-if="filteredVerificationItems.length" class="verification-item-list" ref="results">
-            <VerificationItem
-                v-for="verificationItem in filteredVerificationItems"
-                :key="verificationItem.hash"
-                :verificationItem="verificationItem"/>
-        </div>
+    <div class="certifaction-verification"
+         @dragover.prevent="dragOver"
+         @dragleave="dragLeave"
+         @drop="handleDrop">
 
         <VerificationDropBox
-            @files-dropped="verify"
+            :class="{hidden: !dropbox.draggingOver}"
+            :first-verification="filteredVerificationItems.length === 0"
+            @files-selected="verify"
             @drop="drop"/>
 
-        <div class="powered-by">
-            <span class="label">{{ _$t('verification.poweredBy.label') }}</span>
-            <a href="https://certifaction.io" target="_blank">
-                <img src="../assets/img/certifaction_logo.svg" alt="Certifaction"/>
-            </a>
+        <div :class="{hidden: dropbox.draggingOver}">
```

Comment:
I don't think you need to hide this, just make sure that the dropbox is in front of it and has a background-color.

Reply:
I prefer to hide because when there are a lot of verified items the window is scrollable and i don't think a scrollbar is nice on the dropbox view

Reply:
Ok makes sense. This could be solved by `position: fixed;` on the dropbox, but hiding it is probably even better. It can make the content jump, because either the scrollbar is shown/hidden, or if the parent doesn't have 100% height.